src/number_theory/number_field/embeddings.lean

```diff
@@ -123,3 +125,127 @@ end
 end bounded
 
 end number_field.embeddings
+
+section place
+
+variables {A : Type*} [normed_division_ring A] {K : Type*} [field K] (φ : K →+* A)
+
+/-- An embedding into a normed division ring defines a place of `K` -/
+def number_field.place : absolute_value K ℝ :=
+{ to_fun := norm ∘ φ,
+  map_mul' := by simp only [function.comp_app, map_mul, norm_mul, eq_self_iff_true, forall_const],
+  nonneg' := by simp only [norm_nonneg, forall_const],
+  eq_zero' := by simp only [norm_eq_zero, map_eq_zero, forall_const, iff_self],
+  add_le' := by simp only [function.comp_app, map_add, norm_add_le, forall_const], }
```

Comment:
Is this true more generally as `absolute_value.comp` on `is_absolute_value.to_absolute_value norm`?

Reply:
I added `absolute_value.comp` as you suggested. I thought a long time about where to put it (since `algebra.order.absolute_value` doesn't know about `ring_hom`). Finally, I decided to put it in `topology.algebra.ring` since it is probably the file with the least imports where it fits. I also added `normed_division_ring.is_absolute_value` and used it to prove the already existing `normed_field.is_absolute_value`.